### ISSUE TITLE
Fractional millisecond values are not sanitized or serialized properly on trackDependency

### ISSUE THREAD

**Issue by itilitymaarten** [Microsoft/ApplicationInsights-node.js#270]
```
context.log(`Dependency tracked: ${dependencyName} ${prop} ${duration[0] * 1000 + duration[1] / (1000 * 1000)} false`);
insightsClient.trackDependency(dependencyName, prop, duration[0] * 1000 + duration[1] / (1000 * 1000), false);
```

I use the code above to log a dependency call. The duration array is a result from `process.hrtime()` (the first element contains seconds, the second element contains nanoseconds which couldn't fit into seconds precision). I know the function is called, because the context.log above it is logged (with proper arguments). However, the dependency call isn't showing up in Application insights, while the tracing and everything else I'm logging through the SDK is.

Any suggestions on how to debug this problem or what might be causing it?

**Comment by OsvaldoRosado** [comment]
I can reproduce this problem. The following error is returned from the backend:
`Field 'duration' on type 'RemoteDependencyData' is not a valid duration string. Expected: dd.hh:mm:ss.fffffff format not exceeding 1000 days, Actual: 21.17:03:44.372.2452609539032`

You can workaround this issue by not giving non-integer millisecond values to trackDependency. For example, the following works:
```
var duration = process.hrtime();
client.trackDependency('testDep', 'prop', Math.round(duration[0] * 1000 + duration[1] / (1000 * 1000)), false);
```

**Comment by OsvaldoRosado** [comment]
Fixed with #274
